OpenEXR/IlmImf/ImfTileOffsets.cpp

```diff
@@ -279,7 +279,7 @@ TileOffsets::writeTo (OPENEXR_IMF_INTERNAL_NAMESPACE::OStream &os) const
     
     Int64 pos = os.tellp();
 
-    if (pos == -1)
+    if (pos == Int64(-1))
```

Comment:
static_cast<Int64>

Reply:
I've converted all constructor casts to static_cast